integration/query_comparision_test.go

```diff
@@ -189,6 +189,10 @@ func TestQueryComparisonEq(t *testing.T) {
 			filter:      bson.D{{"value", bson.D{{"$eq", math.NaN()}}}},
 			expectedIDs: []any{"double-nan"},
 		},
+		"DoubleBigInt64": {
+			filter:      bson.D{{"value", bson.D{{"$eq", float64(2 << 61)}}}},
```

Comment:
We should also test with `float64(2 << 61 +1)` to check that we don't get anything, just like we do with Int64DoubleBigPlusOne below

Reply:
I already answered on this question to @ribaraka. 
"It doesn't make sense because a float of this size rounds up a lot and there is no difference between float64(2 << 61) and float64(2 << 61 + 1) and even with adding 100."

Reply:
Sure, and that's exactly what we want to try with both MongoDB and FerretDB to ensure that they behave exactly the same. MongoDB does many things that do not make sense 😆

Reply:
Ok, fixed